README.md

```diff
@@ -4,6 +4,13 @@ sktm - skt manager
 sktm is an orchestrator for skt keeping track of known baselines and tested
 patches.
 
+Installation
+------------
+
+Install `sktm` using pip:
+
+    pip install git+https://github.com/rh-fmk/sktm
+
```

Comment:
This commit ("Documentation updates for pip installs") is fine in itself, but I have an issue with the commit message. Can I ask you to please use imperative mood when writing commit subjects and messages, and not mention "this patch" or "this change", in the future? Using "tags" could save space in subjects as well. E.g. this one could be phrased like this:

    README.md: Add pip install instructions

    Add documentation on installing sktm via pip to README.md.

Another thing which could have been done better, is having the executable name updated in documentation in the same commit with the change itself.